plugins/catalog-backend/src/ingestion/processors/CodeOwnersProcessor.ts

```diff
@@ -37,13 +34,14 @@ const ALLOWED_LOCATION_TYPES = [
   'gitlab/api',
```

Comment:
Gonna need to expand this to work with `url` locations. We've somewhat lost the ability to get straight up integration "type" though. A way forward could be to read the `integrations` config, but we might want to provide some utilities around that from `@backstage/backend-common`.

Reply:
yeah, you are right, there is something missing. I just wanted to migrate it to continue with #2806